comps/comps-katello-client-fedora27.xml

```diff
@@ -19,7 +19,8 @@
       <packagereq type="default">katello-host-tools</packagereq>
       <packagereq type="default">python3-gofer</packagereq>
       <packagereq type="default">python3-gofer-proton</packagereq>
-      <packagereq type="default">python-gofer</packagereq>
+      <packagereq type="default">python2-gofer</packagereq>
```

Comment:
python-gofer == python2-gofer afaik

Reply:
I don't see it in the repo which makes me think this has to be explicitly named -- http://koji.katello.org/releases/yum/katello-nightly/client/f27/x86_64/

Reply:
That's because I just added that packagereq in my last PR unless you've already updated the repos?